husky_components/datagrid/main.js

```diff
@@ -2311,6 +2324,11 @@
                             searchFields: this.options.searchFields.join(',')
                         });
                     }
+                    if (searchString !== '') {
+                        this.storage.set('url', url);
+                    } else {
+                        this.storage.remove('url');
```

Comment:
Or is it not that bad, because a new URL will be generated immediately? That's my impression after testing it.